RecoLocalTracker/SiPhase2Clusterizer/interface/Phase2TrackerClusterizerAlgorithm.h

```diff
@@ -0,0 +1,34 @@
+#ifndef RecoLocalTracker_SiPhase2Clusterizer_Phase2TrackerClusterizerAlgorithm_h
+#define RecoLocalTracker_SiPhase2Clusterizer_Phase2TrackerClusterizerAlgorithm_h
+
+#include "DataFormats/Common/interface/DetSetVector.h"
+#include "DataFormats/Common/interface/DetSetVectorNew.h"
+#include "DataFormats/Phase2TrackerDigi/interface/Phase2TrackerDigi.h"
+#include "DataFormats/Phase2TrackerCluster/interface/Phase2TrackerCluster1D.h"
+
+#include "Geometry/TrackerGeometryBuilder/interface/PixelGeomDetUnit.h"
+
+#include "RecoLocalTracker/SiPhase2Clusterizer/interface/Phase2TrackerClusterizerArray.h"
+
+class Phase2TrackerClusterizerAlgorithm {
+
+    public:
+
+        Phase2TrackerClusterizerAlgorithm(unsigned int, unsigned int);
+        void setup(const PixelGeomDetUnit*);
+        void clusterizeDetUnit(const edm::DetSet< Phase2TrackerDigi >&, edmNew::DetSetVector< Phase2TrackerCluster1D >::FastFiller&);
```

Comment:
here and in a few more places which already have Phase2TrackerCluster1D included
may want to use ``` Phase2TrackerCluster1DCollectionNew ``` instead of 
``` edmNew::DetSetVector< Phase2TrackerCluster1D >```